### PR TITLE
Solid refactoring

### PR DESCRIPTION
**Pull request by Estete9**
### Project requirements

- Make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected.
- Follow SOLID principles.
- Pay special attention to the "Single-responsibility Principle".
  - Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).
  - Extract some pieces of logic into separate class(es).

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -4,119 +4,19 @@
...
+  include CreateInstancesModule
+  include MenuHandlerModule
   attr_accessor :books, :people, :rentals
 
   def initialize
     @books = []
     @people = []
     @rentals = []
   end
-
```
Comment (Shedrack-Sunday) [inline app.rb:16]: - [ ] I was not able to run your app due to naming convention which i corrected, and got it working 👍 .
- Kindly look at the error prompt below, and make the appropriate changes 

### Error and rectified view:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Estete9 @eamador1  @joramot       👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with ruby features ✔️
- Great work on the KISS implementation👏 
- Kudus personalizing the design   ✔️  



# - Required Changes  🌵  

- [ ] I made a couple of comments regarding some features you missed,. Kindly look into it in the comments

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by eamador1** [comment]
Since the last code review, the following changes were made
--

- fix typo when naming decorator file

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED :green_circle: 
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct GitFlow


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**